examples/gno.land/p/demo/uassert/helpers.gno

```diff
@@ -49,3 +49,156 @@ func checkDidPanic(f func()) (didPanic bool, message string) {
 	didPanic = false
 	return
 }
+
+func compare(t TestingT, e1, e2 interface{}, op string) bool {
```

Comment:
- Instead of taking in a `op` parameter, make this function return an `int`: -1 for e1 < e2, 0 for e1 == e2, 1 for e1 > e2.
- you should support strings as well.

Reply:
https://github.com/gnolang/gno/pull/2547/commits/3589a191c99ed5f9f4c9d6f4a0606e209ef9483a